app/src/main/java/woowacourse/paint/painting/figure/Eraser.kt

```diff
@@ -0,0 +1,46 @@
+package woowacourse.paint.painting.figure
+
+import android.graphics.Paint
+import android.graphics.Path
+
+class Eraser(
+    override val path: Path,
+    override val paint: Paint
+) : Figure {
+
+    val isOverlapped: (otherPath: Path) -> (Boolean) = { otherPath ->
+        val region = Path(path).apply {
+            close()
+        }
+        region.op(
+            Path(otherPath).apply {
+                close()
+            },
```

Comment:
```suggestion
            otherPath.apply {
                close()
            },
```
이렇게 수정해도 지우개가 잘 작동되고 있는 것 같은데, 혹시 새로운 Path 객체 안에 넣어 준 후 close를 하는 이유가 있나요?

Reply:
![예시](https://github.com/woowacourse/android-paint/assets/88770426/81f51ea9-effb-429a-aa8d-f4f0a2677f0c)
만약 otherPath를 직접 `close()` 해버리면 다음과 같이 erasing하는 작업 중 내가 그린 line의 본체(?)가 close되어 임시 객체를 만들었다고 생각해주시면 좋을 것 같아요!

Reply:
오~ 저런 이유가 있었군요!? 이해했습니다👍